src/index.html

```diff
@@ -17,6 +30,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
The `href` attribute is set to `#`. Please replace it with an actual URL to ensure proper navigation functionality.